crates/re_data_store/src/store_helpers.rs

```diff
@@ -109,11 +112,12 @@ impl DataStore {
         &self,
         entity_path: &EntityPath,
         query: &LatestAtQuery,
-    ) -> Option<VersionedComponent<C>> {
+    ) -> Option<(Option<TimeInt>, VersionedComponent<C>)> {
```

Comment:
when would this return a component, but not a time?

Could the time be added to `VersionedComponent` instead?